src/manifold/src/smoothing.cpp

```diff
@@ -632,9 +632,10 @@ Vec<Barycentric> Manifold::Impl::Subdivide(
                const int v1 = Next3(v0);
                const int tri = edge.halfedgeIdx / 3;
                for (int i = 0; i < n; ++i) {
-                 glm::vec3& uvw = vertBary[offset + i].uvw;
+                 glm::vec3 uvw(0);
```

Comment:
I don't think the previous behavior is ub? The `vertBary` vector should be initialized...

Reply:
It appeared to be initialized with `-1`s, but only after the first run (it was initialized with `0`s the first time). Not sure why.